### PR TITLE
add node info message #3748

### PR DESCRIPTION
**Pull request by envestcc**

### CODE REVIEW

**Comment by envestcc** [comment]
related to PR https://github.com/iotexproject/iotex-core/pull/3744

File: golang/utils.go [hunk 1]
```diff
@@ -29,6 +29,10 @@ func GetTypeFromRPCMsg(msg proto.Message) (iotexrpc.MessageType, error) {
 		return iotexrpc.MessageType_CONSENSUS, nil
 	case *testingpb.TestPayload:
 		return iotexrpc.MessageType_TEST, nil
+	case *iotextypes.RequestNodeInfoMessage:
+		return iotexrpc.MessageType_NODE_INFO_REQUEST, nil
+	case *iotextypes.ResponseNodeInfoMessage:
+		return iotexrpc.MessageType_NODE_INFO, nil
```
Comment (dustinxie) [inline golang/utils.go:33]: move this to before `MessageType_TEST`

File: golang/utils.go [hunk 2]
```diff
@@ -48,6 +52,10 @@ func TypifyRPCMsg(t iotexrpc.MessageType, msg []byte) (proto.Message, error) {
 		m = &iotextypes.Action{}
 	case iotexrpc.MessageType_TEST:
 		m = &testingpb.TestPayload{}
+	case iotexrpc.MessageType_NODE_INFO_REQUEST:
+		m = &iotextypes.RequestNodeInfoMessage{}
+	case iotexrpc.MessageType_NODE_INFO:
+		m = &iotextypes.ResponseNodeInfoMessage{}
```
Comment (dustinxie) [inline golang/utils.go:56]: move this to before `MessageType_TEST`

File: proto/types/node.proto [hunk 3]
```diff
@@ -20,3 +22,18 @@ message ServerMeta {
...
+    string address = 4;
+}
+
+message ResponseNodeInfoMessage {
+    NodeInfo info = 1;
+    bytes signature = 2;
+}
+
+message RequestNodeInfoMessage {
+}
```
Comment (dustinxie) [inline proto/types/node.proto]: clear the missing/extra line mark

**Review by dustinxie — APPROVED** [review]

**Pull request merged**